snowflake_blueprints/upload_file.py

```diff
@@ -93,6 +93,7 @@ def map_snowflake_to_pandas(snowflake_data_types):
         'REAL': 'float32',
         'DATE': 'datetime64[ns]',
         'TIME': 'datetime64[ns]',
+        'DATETIME' : 'datetime64[ns]',
         'TIMESTAMP': 'datetime64[ns]',
```

Comment:
For this bug fix it doesn't matter but there are additional formats we may want to include for example TIMESTAMP_TZ: https://docs.snowflake.com/en/sql-reference/intro-summary-data-types